crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__enum.snap

```diff
@@ -206,5 +206,99 @@ error[Compatibility E02001]: variant mismatch
    = Enums are part of a module's public interface and cannot be changed during an upgrade.
    = Restore the original enum's variants for enum 'EnumChangeAndAddVariantWithTypes' including the ordering.
 
+error[Compatibility E02001]: variant mismatch
+   ┌─ /fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:78:17
+   │
+78 │     public enum EnumChangeAndRemoveVariantWithPositionalTypes {
+   │                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Missing variant 'C'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's variant 'C' for enum 'EnumChangeAndRemoveVariantWithPositionalTypes' including the ordering.
+
+error[Compatibility E02001]: variant mismatch
```

Comment:
Are these compatibility error codes actually available somewhere to browse?

Reply:
nope not yet, I think we need a plan across all the error codes but for now I'm just following the patterns in the move compiler and adding some codes we can document at some point.